src/components/SearchButton/SearchButton.js

```diff
@@ -57,22 +60,27 @@ class SearchButton extends React.Component {
   renderSearchButton() {
     const classes = cx({ active: this.state.active });
 
+    const label = this.state.active ? 'Close' : 'Search';
+    const iconClass = this.state.active ? XIcon : SearchIcon;
+    const icon = React.createElement(iconClass, {
```

Comment:
Since we are already using `jsx` syntax to declare & import components... React.createElement will be automatically done for us upon transpilation. -- https://reactjs.org/docs/jsx-in-depth.html

To stay consistent I would propose having only one `this.state.active` check/if-condition that will update the label, classes, iconClass and svgIcon... removing the two ternary conditions on `this.state.active` and using let assignment for default values... re-assign with active.